root/etc/cont-init.d/98-ros2

```diff
@@ -0,0 +1,19 @@
+!/usr/bin/with-contenv bash
+
+
+if ! dpkg -l | grep gnupg > /dev/null; then
+    apt-get update && apt-get install -y gnupg
+fi
+
+if [ ! -f "/etc/apt/sources.list.d/ros2.list" ]; then
+    echo "**** Adding ROS2 packages to install list ****"
+    curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+    echo "ros-humble-ros-base ros-dev-tools" >> /mod-repo-packages-to-install.list
```

Comment:
v2 doesn't use the package install script later on so this line should do the `apt install` instead